medschool/src/main.rs

```diff
@@ -192,8 +192,11 @@ impl Display for PartialType {
         let docs = self.docs.last().cloned().unwrap_or_default();
         f.write_str(&docs)?;
 
-        let fields: String = self.fields.iter().map(|field| format!("{field}")).collect();
-        f.write_str(&fields)
+        self.fields.iter().fold(f, |accum, s| {
+            accum.write_str(&s.to_string()).expect("writing failed");
+            accum
+        });
+        Ok(())
```

Comment:
Would this work? So that we can avoid that scary `expect`?
```suggestion
        self.fields.iter().try_for_each(|s| {
            f.write_str(&s.to_string())
        })
```

Reply:
It's in medschool and I think it's better to crash if a field can't be translated to str then to quietly fail